index.js

```diff
@@ -25,14 +26,16 @@ module.exports = (input, opts) => {
 	input = input || 'patch';
 
 	opts = Object.assign({
-		cleanup: true
+		cleanup: true,
+		yarn: !opts.noYarn && hasYarn()
```

Comment:
I just realized that `--no-yarn` gets parsed as `opts.yarn`, was sure it was parsed as `opts.noYarn`.

CI will fail anyways because `opts` is `undefined` on the tests. Will check why that happens.